crates/provider/src/ext/trace.rs

```diff
@@ -10,9 +10,6 @@ use alloy_rpc_types_trace::{
 };
 use alloy_transport::{Transport, TransportResult};
 
-/// List of trace calls for use with [`TraceApi::trace_call_many`]
-pub type TraceCallList<'a, 'b, N> = [(&'a <N as Network>::TransactionRequest, &'b [TraceType])];
```

Comment:
why remove this alias?

Reply:
Don't really see a need for it but adding it back to prevent adding unnecessary breaking changes